biscuit.gemspec

```diff
@@ -13,15 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = %q{Ruby wrapper for biscuit (https://github.com/dcoker/biscuit).}
   spec.homepage      = "https://github.com/usertesting/biscuit"
 
-  # Prevent pushing this gem to RubyGems.org by setting 'allowed_push_host', or
-  # delete this section to allow pushing this gem to any host.
-  if spec.respond_to?(:metadata)
-    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
-  else
-    raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
-  end
-
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features|doc)/}) }
```

Comment:
we probably ought to specify the license in here too, and update the authors/email (suan doesn't have write-access anymore)